### DISCUSSION TITLE
Please remove pop up notification when toggling copilot on/off

### DISCUSSION THREAD

**Discussion by rileyblackwell** [community/community#50738] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

It would be more efficient to allow copilot to toggle on/off for the current workspace without needing to click "disable globally" or "disable for the current language". Just a simple keyboard shortcut without having to click the mouse would be great! Similar to how you can toggle single lines comments with "ctrl /" keyboard shortcut.

**Comment by zkrzyzanowski** [comment]
seconding this feedback, copilot isn't quite ready to be left on full time as it still conflicts with intellisense. I ended up canceling my subscription too it because toggling it on and off takes too much time with the warning. Otherwise, I'd love to use it.
